Components/Core/Inc/Timer.hpp

```diff
@@ -7,23 +7,18 @@
 #ifndef AVIONICS_INCLUDE_SOAR_CORE_TIMER_H
 #define AVIONICS_INCLUDE_SOAR_CORE_TIMER_H
 /* Includes ------------------------------------------------------------------*/
-#include "cmsis_os.h"
-#include "Utils.hpp"
 #include "FreeRTOS.h"
+#include "Utils.hpp"
+#include "cmsis_os.h"
 
 /* Macros --------------------------------------------------------------------*/
-constexpr uint32_t DEFAULT_TIMER_COMMAND_WAIT_PERIOD = MS_TO_TICKS(15); // Default time to block a task if a command cannot be issued to the timer
+constexpr uint32_t DEFAULT_TIMER_COMMAND_WAIT_PERIOD =
+    MS_TO_TICKS(15);  // Default time to block a task if a command cannot be issued to the timer
 
-#define DEFAULT_TIMER_PERIOD (MS_TO_TICKS(1000)) // 1s
+#define DEFAULT_TIMER_PERIOD (MS_TO_TICKS(1000))  // 1s
 
 // Enumeration representing the 4 timer states
-enum TimerState {
-    UNINITIALIZED=0,
-    COUNTING,
-    PAUSED,
-    COMPLETE
-};
-
+enum TimerState { UNINITIALIZED = 0, COUNTING, PAUSED, COMPLETE };
```

Comment:
This should be changed, enums should be multi-line since we specify specific values often.